lib/common/http/downstream-post-stream-to-server.ts

```diff
@@ -52,7 +52,7 @@ class BrokerServerPostResponseHandler {
     this.#serverId = serverId;
     this.#requestId = requestId;
     this.#buffer = new stream.PassThrough({ highWaterMark: 1048576 });
-
+    this.#brokerSrvPostRequestHandler;
```

Comment:
@aarlaud, we don't instantiate this variable here, do we need this line?

Reply:
updated, sorry